### PR TITLE
bridge-method-annotation should be an optional dep

### PR DESCRIPTION
**Pull request by jglick**
b6e48cc4f9c7b13895673a7b4aa6399b7e236b1f introduced a hard dep on a lib which is newer than that currently bundled in core, causing problems for `requireUpperBoundDeps`.

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -131,6 +131,7 @@
       <groupId>com.infradna.tool</groupId>
       <artifactId>bridge-method-annotation</artifactId>
       <version>1.14</version>
+      <optional>true</optional>
```
Comment (KostyaSha) [inline pom.xml]: or scope provided?

Comment (jglick) [reply pom.xml]: Either seem to work pretty much the same.

**Comment by vivek** [comment]
LGTM 🐝

**Comment by recampbell** [comment]
@reviewbybees complete

**Pull request merged**